src/System.Management.Automation/engine/CommonCommandParameters.cs

```diff
@@ -232,6 +232,22 @@ public string PipelineVariable
             set { _commandRuntime.PipelineVariable = value; }
```

Comment:
@KirkMunro, your last commit had 2 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By CustomPipeName.Can enter, exit, and re-enter using CustomPipeName
```powershell
Expected $true, because The script was able to re-enter another process and grab the pipe of 'jnbdou4t.bj4'., but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 170
170:                     Should -BeTrue -Because "The script was able to re-enter another process and grab the pipe of '$pipeName'."
```
Invoke-WebRequest tests.Denial of service.Image Parsing
```powershell
Expected the actual value to be greater than 10, but got 6.0951514218109.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\WebCmdlets.Tests.ps1: line 1921
1921:             $pathologicalRatio | Should -BeGreaterThan 10
```

Reply:
@PoshChan please restart Windows

Reply:
@PoshChan please restart Windows